src/components/UriViewer/UriViewer.js

```diff
@@ -147,10 +209,7 @@ export const UriViewer = _.flow(
         ]),
         div({ style: { fontSize: 10 } }, ['* Estimated. Download cost may be higher in China or Australia.'])
       ])],
-      () => h(Fragment, [
-        isGsUri(uri) ? 'Loading metadata...' : 'Resolving DRS file...',
-        spinner({ style: { marginLeft: 4 } })
-      ])
+      LoadingSymbol(uri)
```

Comment:
Each argument to Utils.cond should be either an array of `[boolean, function]` or a function.